docker/index.html

```diff
@@ -0,0 +1,10 @@
+ <!DOCTYPE html>
+ <html>
+  <head>
+    <title>Docker</title>
+    <meta http-equiv="refresh" content="0;URL='https://hub.docker.com/search?q=owasp%2Fzap&type=image'" />
+  </head>
+  <body>
+    <p>Redirecting</p>
```

Comment:
Do we want to URL to show in the body content? I can add it, was just trying to minimize future maint. changes.

Reply:
Should be fine as is.